### ISSUE TITLE
Contents of some menu dropdowns cannot be accessed with keyboard

### ISSUE THREAD

**Issue by enriquesanchez** [ParabolInc/action#6697]
On some menu items, I can open the the dropdown with my keyboard but once opened, I cannot access the contents of them. 

Hitting <kbd>ESC</kbd> closes the menu and returns my focus to the toggle as expected. Hitting <kbd>TAB</kbd> returns my keyboard focus to the start of the page, which is disorienting. 

<img width="300" alt="Captura de Pantalla 2022-06-08 a la(s) 10 41 49 a m" src="https://user-images.githubusercontent.com/3276087/172660103-9f85c94b-3991-4589-bb63-86624bd75654.png">

<img width="225" alt="Captura de Pantalla 2022-06-08 a la(s) 10 41 58 a m" src="https://user-images.githubusercontent.com/3276087/172660127-5180b5b1-caef-40cc-b51d-8695b00528f5.png">

## Acceptance criteria

- [ ] All menu items can be fully operated with a keyboard
- [ ] Keyboard focus is retained as expected

### Estimated effort

- 1 hr for identifying issue
- 1 hr for implementing fix
- 1 hr for review
Total: 3 hrs
